### PR TITLE
standardized the module

### PR DESCRIPTION
**Pull request by maxired**
Hi,

Here is a try to standardized the module with others;
Changes are : 
 - move main file to lib/index.js
 - update dependencies
 - use lab instead of mocha for test
 - respect Standard syntax

### CODE REVIEW

**Comment by mcdonnelldean** [comment]
@maxired can you kill your node_modules and rerun your tests? Looks like a missing link somewhere.

**Comment by maxired** [comment]
Hi,
sorry about that. I am currently trying to solve the issue.

Looks like the problem appears after moving the main file to lib/index.js

**Comment by mcdonnelldean** [comment]
@maxired I best the issue is in how seneca-transport-test is loading the module

**Comment by maxired** [comment]
An the new error is because beanstalkd is not running in the Travis instance

**Comment by mcdonnelldean** [comment]
@maxired Interesting, let's pull in @geek see if he can save they day :D

**Comment by maxired** [comment]
Unfortunately, there is no way to speed up the travis build using container, because beanstalkd doesn't comply to travis container limitation. see -> https://github.com/travis-ci/apt-package-whitelist/issues/780

File: package.json [hunk 1]
```diff
@@ -18,18 +18,20 @@
...
+   "fivebeans": "~1.3.1",
+    "lodash": "^3.10.1",
+    "nid": "~0.3.2"
   },
   "devDependencies": {
-    "mocha": "~2.1.0",
+    "eslint-config-standard": "^4.4.0",
+    "eslint-plugin-standard": "^1.3.1",
+    "lab": "^6.2.0",
     "seneca": "plugin",
```
Comment (maxired) [inline package.json]: @geek , @mcdonnelldean any thought about  `"seneca" : "plugin"` ? 

Should we use peerDependencies instead ?

Comment (geek) [inline package.json]: You should use both dev and peer for seneca.


dev will install seneca for test purposes.
peer will ensure that seneca can be installed alongside beanstalk-transport

**Comment by maxired** [comment]
@mcdonnelldean , I believe this is now good enough to be merged, what you think ?

**Comment by mcdonnelldean** [comment]
@maxired Looks good :dancer:

**Pull request merged**